packages/shared/src/hooks/useTokenSecurity.ts

```diff
@@ -0,0 +1,21 @@
+import { GoPlusLabs, SecurityAPI } from '@masknet/web3-providers'
+import { ChainId, ZERO_ADDRESS } from '@masknet/web3-shared-evm'
+import { first, isEmpty } from 'lodash-unified'
+import { useAsyncRetry } from 'react-use'
+
+type TokenSecurity = SecurityAPI.ContractSecurity &
+    SecurityAPI.TokenSecurity &
+    SecurityAPI.TradingSecurity & { contract: string; chainId: ChainId }
+
+export const useTokenSecurity = (chainId: ChainId, address?: string, isTokenSecurityClosed = false) => {
+    return useAsyncRetry(async () => {
+        if (isTokenSecurityClosed) return
+        if (!address || address === ZERO_ADDRESS) return
```

Comment:
```suggestion
        if (isSameAddress(address, ZERO_ADDRESS)) return
```

Reply:
when address is ```undefined```, ```isSameAddress``` function returns false